src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -36,17 +36,18 @@ public sealed partial class JsonArray : System.Text.Json.JsonNode, System.Collec
         public JsonArray(System.Collections.Generic.IEnumerable<ulong> values) { }
         public int Count { get { throw null; } }
         public bool IsReadOnly { get { throw null; } }
+        [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
         public System.Text.Json.JsonNode this[int idx] { get { throw null; } set { } }
```

Comment:
I'm confused as to what this attribute does/means here.

Reply:
It means that the setter accepts null (but the getter won't return null).